athena/loss.py

```diff
@@ -97,3 +97,77 @@ def __call__(self, logits, samples, logit_length=None):
         loss = tf.reduce_sum(tf.abs(loss, name="L1_loss"), axis=-1)
         loss = tf.reduce_mean(loss)
         return loss
+
+def GeneratorLoss(discirmination, input_real, generated_back,identity_map,\
+                                           target_label_reshaped, domain_out_real):
+    lambda_cycle = 10
```

Comment:
make these hyperparams or at least write some explanation for these magic number